src/Symfony/Bridge/Monolog/Processor/WebSubscriberProcessor.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Processor;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+if (!interface_exists(EventSubscriberInterface::class)) {
+    throw new \RuntimeException('The "WebSubscriberProcessor" requires the EventDispatcher component. Please run "composer require symfony/event-dispatcher".');
+}
+
+/**
+ * WebProcessor override to read from the HttpFoundation's Request and to be autoconfigured.
```

Comment:
Sorry, but I don't understand what that means as well (I see that you copy/pasted the one from WebProcessor, but the second part is quite cryptic to me).